src/widgets/D2ContentEditable.js

```diff
@@ -1,62 +0,0 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import ContentEditable from 'react-contenteditable';
```

Comment:
Make sure to remove this dependency as well.

Reply:
Thanks